.github/workflows/gitpod-web-docker.yml

```diff
@@ -1,37 +1,47 @@
 name: Gitpod-Web Docker
 on:
   workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Not sure if we should push a new image for every new commit we do :thinking:

Reply:
I'll remove this to publish manually